source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.
+
+Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).
```

Comment:
Should we be consistent in calling the upstream either the "Pantheon Drupal 7 Upstream" or "drops-7" vs switching?